src/test_utils.rs

```diff
@@ -702,6 +705,25 @@ impl TestContext {
             );
         }
     }
+
+    pub async fn create_group_with_members(
```

Comment:
Can we have doc comments?  I love doc comments, especially good ones that say something new and helpful.

Reply:
There aren't many new and helpful things the doc comment could say, only maybe that it'll stay unpromoted. Like, `Create a new group with the given members. The group will be unpromoted, i.e. to make the other members aware of it, you need to send a message to it and let the others receive it.`